sktime/utils/_maint/tests/test_show_versions.py

```diff
@@ -26,8 +26,12 @@ def test_deps_info():
     assert isinstance(deps_info_default, dict)
     assert set(deps_info_default.keys()) == set(DEFAULT_DEPS_TO_SHOW)
 
+    PKG_IMPORT_ALIAS = {"scikit-learn": "sklearn"}
+
     for key in DEFAULT_DEPS_TO_SHOW:
-        key_is_available = _check_soft_dependencies(key, severity="none")
+        key_is_available = _check_soft_dependencies(
+            key, severity="none", package_import_alias=PKG_IMPORT_ALIAS
```

Comment:
@fkiraly here `key` is `sklearn`, as that's what mentioned here:

https://github.com/sktime/sktime/blob/8bd40f16c35a5a1894b85c31600201120655c350/sktime/utils/_maint/_show_versions.py#L40

However, `_check_soft_dependencies` expect `scikit-learn` as the input, not `sklearn`.

Reply:
Yes, I think that should be ok, because `_get_deps_info` uses the import path, not the environment specs.

Reply:
But this is causing `key_is_available` to be `False` in case of `scikit-learn`, do we want that? Is it not expected to be `True` for installed packages?

What is your expected value for this?

> _check_soft_dependencies("scikit-learn", severity="none", package_import_alias={'scikit-learn':"sklearn"})

On `main`, it is `True`, on this branch it is `False`.

Reply:
oh, I see! That's the mistake.

Reply:
attempted fix

Reply:
I made a commit from Github UI, hopefully you don't mind.